input_topic: "/camera/depth/points"
x_vel: 0.1
min_dist: 1.0